Formulaic TOS

```diff
@@ -0,0 +1,73 @@
+By accepting these terms, you also accept the Formulaic Privacy Notice and Formulaic Content Policies.
+
+###A. Creating an Account
+In order to view or run shared queries, or to share content of your own, you will need to create an account on the Service using your Google Account.  You can create a Google Account on [Google’s signup page](https://accounts.google.com/lifecycle/steps/signup/name?dsh=S1694799353:1707861924262237&flowEntry=SignUp&flowName=GlifWebSignIn&theme=glif&TL=ADg0xR0JpndiwBwSSLq5Ih_cBj1vPyVBRu2lVXVKxOqIHKvLx__ThXaRX7v0OVnt).  More information about the information that we receive from Google, and how we use that information, is included in the Formulaic Privacy Notice. Google Accounts has it’s own [Privacy Notice](https://policies.google.com/privacy?hl=en-US).
+
+###B. Using the Service
+You give Mozilla all rights necessary to operate the Service, including processing the data you upload to or access through the Service such as queries & results, “saves,” user profiles, and similar items, as referenced in our Privacy Notice.  
+You will comply with Mozilla’s [Acceptable Use Policy](https://www.mozilla.org/about/legal/acceptable-use/) as well as the other policies referenced in this Agreement.  You also agree that you will not use the Service to violate any law in the jurisdiction where you are based, or to infringe the rights of one or more third parties.   
+You will use the service only for the purpose of creating, sharing, exchanging, testing and saving LLM prompts and their resulting responses.  If you appear to be using the Service for something other than its intended purpose, including merely to access the underlying AI models, your use may be terminated.  You may not use the service, or any of the underlying LLM models, to support the creation of your own LLM.  You also may not use the service to create a product that competes with any product developed by Google.
+Any ideas, suggestions, or feedback that you provide about the Service may be used by Mozilla for free, without any additional obligations.
+When you use the Service to run a query, the response will be provided by a third party LLM operator.  If you use the Google LLM, Google may use your (anonymized) content to train its AI model.  Outputs from LLMs may not always be accurate, and should not be relied on for factual information, or for advice related to finances, employment, education, housing, legal, medical, career, or other important personal decisions.  LLM outputs are affected by a number of factors such as inputs, training data, programming, and others, and may be incomplete, incorrect, or offensive, or refer to specific products, companies, or individuals.  They do not represent Mozilla’s views, or indicate an endorsement of any kind.
+
+###C. Term; Termination
```

Comment:
What is "Term" in this context?

Reply:
Term = the term (length) of the contract.  The terms will continue to apply for an indefinite term until terminated by one of the parties